common/PROTOC.md

```diff
@@ -3,6 +3,33 @@
 ## Installation
 Run `npm install` to install node modules
 
+## Setting
+A project must contain `proto.json` file with paths to required files.  
+Example:
+```
+{ 
+  // Path to folder, where proto functions will be located.
+  // The path is relative to the project
+  "protoPath": "src/app/core/proto",
+  // List with paths to each .proto file inside the project.
+  // The paths are relative to the project
+  "projectProtoList": [
+    "proto/apples.proto",
+    "proto/pineapples.proto"
+  ],
+  // List with paths to each .proto file inside the repo.
+  // The paths are relative to the repo
+  "repoProtoList": [
+    "common/repo/repo.proto"
+  ]
+}
+```
+
+There is `shared-proto.json`, which is list of paths to each .proto file inside the workspace.  
```

Comment:
Why do we have to declare the proto files in the workspace in advance?

Reply:
`shared-proto.json` contains list of proto, which is added to each project.

Reply:
Always added? 
What if a project doesn't need them?

Reply:
Well, it's optional. We can do not add the file to `shared-proto.json`.
Also, If you think there won't be a file, which is used in every project for sure, then we could remove the list.